sdk/kusto/Azure.ResourceManager.Kusto/src/Generated/Models/FollowerDatabaseDefinitionGet.cs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Collections.Generic;
+
+namespace Azure.ResourceManager.Kusto.Models
+{
+    /// <summary> A class representing follower database object. </summary>
+    public partial class FollowerDatabaseDefinitionGet
```

Comment:
This model name is quite weird, consider a better name determined by the use scenario.

Reply:
changed to KustoFollowerDatabase